integrations/atlassian/jira/event.go

```diff
@@ -75,6 +75,14 @@ func (h handler) handleEvent(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// issue_property_set and issue_property_deleted events aren’t supported or relevant to AutoKitteh.
+	// They occur when custom fields are programmatically changed on an issue.
+	// We ignore them because our webhook only extends expiration and can’t update subscriptions.
+	if jiraEvent["webhookEvent"] == "issue_property_set" || jiraEvent["webhookEvent"] == "issue_property_deleted" {
```

Comment:
Shorter, and good enough:
```suggestion
	if strings.HasPrefix(jiraEvent["webhookEvent"], "issue_property") {
```